test/runtests.jl

```diff
@@ -12,22 +12,6 @@ end
     include("initial_condition_tests/initial_profile_test.jl")
 end
 
-@safetestset "Box Test" begin
-    include("experiments/box_driver/run_box_simulation.jl")
```

Comment:
We don't want those tests anymore?

Reply:
These experiments fit more in buildkite and are called there. I removed them from CI tests to save computations.